src/lib/updating-element.ts

```diff
@@ -793,3 +793,9 @@ export abstract class UpdatingElement extends HTMLElement {
   protected firstUpdated(_changedProperties: PropertyValues) {
   }
 }
+
+/**
+ * Marks class as having finished creating properties.
+ */
+const finalized = new Set<typeof UpdatingElement>();
```

Comment:
Wouldn't a `WeakSet` be better here? As is, this'll strongly hold onto every `LitElement`.

Reply:
This would only hold on to class definitions (`typeof UpdatingElement`). But also, @aomarks is changing the PR to use a property again, just with the index operator and string literal to avoid Closure's bug. That'll allow the GC to collect classes if they every become freeable.

Reply:
Ah, I missed the `static` keyword in the code. 👍